test/connectivity/scripts/connectivityTest.sh

```diff
@@ -499,7 +531,7 @@ function usage() {
     echo ' -storageAccountConnectionString          Azure storage account connection string with privilege to create blob container.'
     echo ' -edgeRuntimeBuildNumber                  Build number for specifying edge runtime (edgeHub and edgeAgent)'
     echo ' -testInfo                                Contains comma delimiter test information, e.g. build number and id, source branches of build, edgelet and images.'
-
+    echo ' -testPlatform                            Test agent platform'
```

Comment:
You don't need test platform parameter as you can use $image_architecture_label variable in connectivity script to decide "optimize_performance_flag".